tagstudio/src/qt/widgets/migration_modal.py

```diff
@@ -655,14 +653,17 @@ def check_subtag_parity(self) -> bool:
 
         with Session(self.sql_lib.engine) as session:
             for tag in self.sql_lib.tags:
+                if tag.id in range(0, 1000):
+                    break
```

Comment:
why `break` here? shouldn't it `continue`?
(same for the other loops in `check_alias_parity`, `check_shorthand_parity`, and `check_color_parity`)